TheFourFifths/consus-client
# Modal style tff211

### Summary

Styling all the modals.  Guys.  There's a lot of modals.

### Checklist


### How to Review

1. There's a lot of modals, so buckle your seatbelts.
2. Think of every error the system produces, then think of 30 more.  Test all of them.
3. Try the date modal for existing->longterm
4. Try the input modals for Admin override, rfid->student association, etc...
5. Try the confirm modals for item deletion, model delete/edit, non-existing rfid, etc...
6. Run tests
7. Find more modals (there are so many modals, guys)

### Links

- [TFF-221](https://msoese.atlassian.net/browse/TFF-221)
